gamedata/tf2.utils.nosoop.txt

```diff
@@ -44,16 +44,16 @@
 					"offset"		"53"
 				}
 			}
-			"offsetof(CTFPlayer::m_flRespawnTimeOverride)"
+			"offsetof(::lagcompensation)"
 			{
-				"signature"		"CTFPlayer::IsReadyToSpawn()"
+				"signature"		"CTFWeaponBaseMelee::Smack()"
 				"linux"
 				{
-					"offset"	"108"
+					"offset"	"29h"
 				}
 				"windows"
 				{
-					"offset"	"129"
+					"offset"	"1Bh"
```

Comment:
It looks like this is supposed to be a new section and not deleting `offsetof(CTFPlayer::m_flRespawnTimeOverride)`?

Reply:
Yea...